### PR TITLE
ParaView: add new ParaView-5.9.0-RC2 release

### PR DESCRIPTION
**Pull request by vicentebolea**
Hi there :)

This evening we have just released the latest ParaView version.

I have made changes in the ParaView Spack package which adds this new version.

Note that this is a release candidate, but we are bringing many changes with this release and we have told to ourselves: why wait to the the final release?

VB

### CODE REVIEW

**Comment by vicentebolea** [comment]
@cquammen

**Comment by danlipsa** [comment]
+1

**Review by cquammen — APPROVED** [review]

**Comment by cquammen** [comment]
@chuckatkins do you know who has write access and can approve this PR?

**Comment by cquammen** [comment]
@adamjstewart could you please review? Thank you.

**Review by adamjstewart — CHANGES_REQUESTED** [review]
You should make the latest stable release `preferred=True`, otherwise the release candidate will be installed by default.

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -19,6 +19,9 @@ class Paraview(CMakePackage, CudaPackage):
 
     maintainers = ['chuckatkins', 'danlipsa']
 
+    # Release Candidate versions
+    version('5.9.0-RC2', sha256='a3e6f7db7804f11351df48781e8d67ff3e9c18a691bb9ef9a5485aa6520e70bc')
+
     version('develop', branch='master', submodules=True)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/paraview/package.py:22]: We should call this master for consistency

Comment (vicentebolea) [reply var/spack/repos/builtin/packages/paraview/package.py:22]: Changed in the last push. 

Thanks for the review!

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 2]
```diff
@@ -19,6 +19,9 @@ class Paraview(CMakePackage, CudaPackage):
 
     maintainers = ['chuckatkins', 'danlipsa']
 
+    # Latest release, it can be either a candidate or a stable release
+    version('master', sha256='a3e6f7db7804f11351df48781e8d67ff3e9c18a691bb9ef9a5485aa6520e70bc')
+
     version('develop', branch='master', submodules=True)
     version('5.8.1', sha256='7653950392a0d7c0287c26f1d3a25cdbaa11baa7524b0af0e6a1a0d7d487d034')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/paraview/package.py]: I think there's been some misunderstanding. What I was suggesting is:
```suggestion
    version('master', branch='master', submodules=True)
    version('5.9.0-RC2', sha256='a3e6f7db7804f11351df48781e8d67ff3e9c18a691bb9ef9a5485aa6520e70bc')
    version('5.8.1', sha256='7653950392a0d7c0287c26f1d3a25cdbaa11baa7524b0af0e6a1a0d7d487d034', preferred=True)
```

**Review by adamjstewart — APPROVED** [review]

**Comment by vicentebolea** [comment]
@adamjstewart 

it seems that the  `ci/gitlab-ci —` has failed in the building stage. I am not able to access the logs. Could you share them to fix this?

Thanks!

**Pull request merged**